### PR TITLE
Implemented individual overview functionality.

### LINKED ISSUE #127: Individual project overviews
### Defining the problem
Currently only one overview can be defined in a project. In the case of a multi project structure this seems like a major limitation, It is possible to define an overview which includes sections for each project  but this is a clunky solution that impairs user experience.

### Proposed solution
I would propose a modification to the docs app that would allow a user to define a docs block in an .md file that by simply using a particular naming convention for example {% docs \<project name>_overview %}. That block would be used in place of the main overview whenever the user clicks on the root folder of the project in the file tree, and/or perhaps adds some tabbed selection area in the overview panel.

### Benefits
This would allow anyone building a project that includes sub-project to easily define more intuitive and extensive documentation.

### PR DESCRIPTION
**Pull request by Mr-Nobody99**
The primary changes are to the routing (added a new route for project_overviews),
and the model tree (added a condition to the click event to route to project overview),
I also added an example docs block in the ci-project/models/README.md

I made the changes in such a way that if there are no project level overview block defined default functionality should be unaffected. (ie. display __overview__ block as normal.)

Fixes #127

### CODE REVIEW

**Comment by drewbanin** [comment]
this is really neat @Mr-Nobody99!! I think we'll need to re-compile the artifacts in the `data` folder to get this working in the [PR build environment](https://deploy-preview-129--eloquent-tesla-2c6e1b.netlify.app/#!/overview/jaffle_shop).

I largely feel pretty good about an approach like this, but I'm tagging @jtcohen6 to see if he has any additional thoughts. I imagine a good scheme being `__{{ project_name }}__` instead of `{{ project_name }}_overview`, but that feels like a pretty minor detail to hammer out.

Nice work!

**Comment by Mr-Nobody99** [comment]
Thank you much @drewbanin, glad to hear that, I look forward to any input @jtcohen6 has as well. I also went ahead and changed the naming format to __{{ project_name }}__ as you suggested.

File: src/app/overview/index.js [hunk 1]
```diff
@@ -11,17 +11,25 @@ angular
...
             var overviews = _.filter(project.docs, {name: '__overview__'});
-            _.each(overviews, function(overview) {
+            _.each(overviews, function (overview) {
                 if (overview.package_name != 'dbt') {
                     selected_overview = overview;
                 }
             });
-
+            if (project_name !== null) {
+                let project_key = `${project_name}.__${project_name}__`;
```
Comment (jtcohen6) [inline src/app/overview/index.js:27]: I think we'll want to let top-level projects "override" the project-level overviews of installed packages. If `my_project` installs the `snowplow` package, I would want `my_project.__snowplow__` to take precedence over `snowplow.__snowplow__`, especially if the latter doesn't exist.

Comment (Mr-Nobody99) [reply src/app/overview/index.js]: Thanks a lot @jtcohen6 I'm glad you like the idea, I agree with your comment completely. I added a new commit that I think should handle that scenario. Let me know if you think it needs any other change.

File: ci-project/models/overview.md [hunk 2]
```diff
@@ -9,3 +9,12 @@ This [dbt](https://www.getdbt.com/) project is for demonstrations and tutorials.
...
 {% enddocs %}
+
+{% docs __jaffle_shop__ %}
+## Jaffle_shop Project level overview example
+This [dbt](https://www.getdbt.com/) project may have only one project (`jaffle_shop`) currently.
+However there may be other projects included in your own projects.
+
+You can assign a unique overview for each project by adding a docs block in an .md file of your project
+and giving it a name with the following convention __<project_name>__.
+{% enddocs %}
```
Comment (jtcohen6) [inline ci-project/models/overview.md:20]: * Could you add an entry for `{% docs __dbt_utils__ %}` below?
* As Drew mentioned, to really see your change in the PR preview, we'll need to run `dbt docs generate` in the `ci-project` to replace the artifacts currently in `data/`. If you're up for doing this as part of this PR, great! If not, we can do it separately after merging.

**Comment by jtcohen6** [comment]
@cla-bot check

File: ci-project/models/overview.md (hunk 2 above)
Comment (Mr-Nobody99) [reply ci-project/models/overview.md:29]: @jtcohen6  Thanks for the suggestion I added a block for dbt_utils. I also generated docs in the ci-project and replaced the data/ files with them. I also signed the CLA. Please let me know if I did anything incorrectly with the data/ artifacts.

File: ci-project/models/overview.md [hunk 3]
```diff
@@ -9,3 +9,21 @@ This [dbt](https://www.getdbt.com/) project is for demonstrations and tutorials.
...
+<<<<<<< HEAD
+and giving it a name with the following convention { \__project_name__ }
+{% enddocs %}
+
+{% docs __dbt_utils__ %}
+## DBT_UTILS package overview
+This [dbt package](https://docs.getdbt.com/docs/building-a-dbt-project/package-management) is a collection of tools to help with common tasks.
+=======
+and giving it a name with the following convention __<project_name>__.
+>>>>>>> parent of b2999b6... Added overview block for dbt_utils, and generated new docs from ci_project(replaced existing in data/)
```
Comment (jtcohen6) [inline ci-project/models/overview.md:28]: Tiny thing: looks like there are a few git vestiges from merging

Comment (Mr-Nobody99) [reply ci-project/models/overview.md]: Ah my bad, next commit should fix.

**Pull request merged**